### ISSUE TITLE
training w/ multi classes from open images dataset 

### ISSUE THREAD

**Issue by faresbs** [qfgaohao/pytorch-ssd#51]
When training on multi-classes (using the downloaded images from open-images V4), the training loss becomes very large and sometimes i get nan. i believe that it is because i am training using classes (Human body and Human hand or Human face) that uses the same image distribution. And since the data for Human body is bigger, my model ends up recognizing only that class. 

Is there a way of downloading images that contains all the bounding boxes of all the wanted classes?

**Comment by qfgaohao** [comment]
something like this 'python open_images_downloader.py --root ~/data/open_images --class_names "Handgun,Shotgun" --num_workers 20'
